### ISSUE TITLE
Weird error with webshot

### ISSUE THREAD

**Issue by brunohr** [odeleongt/postr#13]
First, i'd like to thank for this package.

Sadly, i get a error message every time i try to generate the .png, and webshot returns this message:

`Output created: teste.html
Could not load  file:///C:/Users/playe/Desktop/teste/C:/Users/playe/Desktop/teste/teste.html
Error in webshot::webshot(url = input, file = output, vwidth = width,  : 
  webshot.js returned failure value: 1
Calls: <Anonymous> ... overlay -> print_render -> <Anonymous> -> <Anonymous>
Além disso: Warning messages:
1: In normalizePath(path.expand(path), winslash, mustWork) :
  path[1]="C:/Users/playe/Desktop/teste/./C:/Users/playe/Desktop/teste/teste.html": A sintaxe do nome do arquivo, do nome do diretório ou do rótulo do volume está incorreta
2: execução do comando '"C:\Users\playe\AppData\Roaming/PhantomJS/phantomjs.exe" "C:/Users/playe/Documents/R/win-library/3.4/webshot/webshot.js" "[{\"url\":\"file:///C:/Users/playe/Desktop/teste/C:/Users/playe/Desktop/teste/teste.html\",\"file\":\"C:/Users/playe/Desktop/teste/./C:/Users/playe/Desktop/teste/teste.png\",\"vwidth\":1200,\"vheight\":1697,\"delay\":1,\"zoom\":5.8465}]"' teve status 1 
Execução interrompida`

Basically, it seems that my file path is being called twice (`C:/Users/playe/Desktop/teste/C:/Users/playe/Desktop/teste/teste.html\` instead of just `C:/Users/playe/Desktop/teste/teste.html`). I checked my webshot and the phantomJS, but i can't find a solution

**Comment by odeleongt** [comment]
Hi! Thanks for reporting. I happened to me while I was preparing a poster three weeks ago, and in the rush of printing I forgot to push the fix.
Update the package and try again!
It seems it resulted from sum updates to rmarkdown.
